forms/jqm/element.js

```diff
@@ -68,9 +71,13 @@ define(function (require) {
     renderLabel: function () {
       if (!this.$label) {
         this.$label = $('<label class="ui-input-text"></label>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
You would think so... but no. :(

it was already setting it to null. I just removed it from the DOM before setting it to null, just to be complete. I believe if a view sets up an element and keeps a reference to that element, it should explicitly remove that element and the reference to it.

The strange thing is that the code gets called, but then when you change pages `this.$label` is set to the label that was removed. This is why I think there is something deeper at play - but in the mean time we need to make it look like its working - missing labels easy to spot and make us look bad,